sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/context.hpp>
+#include <azure/core/nullable.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
+#include <azure/core/response.hpp>
+
+using namespace Azure::Core;
+
+namespace Azure { namespace Core { namespace Test {
+
+  class StringClient;
+
+  class StringOperation : public Operation<std::string> {
+
+  private:
+    StringClient* m_client;
+    std::string m_operationToken;
+    std::string m_value;
+
+  private:
+    int count = 0;
+
+  public:
+    StringOperation(StringClient* client){ m_client = client; }
+
+
+    std::string GetResumeToken() { return m_operationToken; }
+
+    std::unique_ptr<Http::RawResponse> Poll(Context& context = Context())
```

Comment:
Missing `#include <memory>`.

Reply:
How do you catch such things in code review @CaseyCarter ?

Reply:
1. Memorize which Standard Library header declares each entity. (This is really a product of decades of experience moreso than rote memorization, at which I'm not great.)
2. Cross-check each library name I run across against the list of headers (which is ideally short enough to memorize) at the beginning of the file.

Ironically I get a little better at (1) and a little worse at (2) every year.